model-optimizer/extensions/middle/FusedBatchNormNonConstant.py

```diff
@@ -40,8 +40,7 @@ def run_before(self):
     def pattern(self):
         return dict(
             nodes=[
-                ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2',
-                                                            'FusedBatchNormV3']))],
+                ('op', dict(kind='op', op=lambda op: op in ['batchNormInference']))],
```

Comment:
Change to `('op', dict(kind='op', op='batchNormInference'))],`

Reply:
FusedBatchNormNonConstant.py  was deleted